Gemfile

```diff
@@ -63,6 +63,9 @@ group :test do
   gem 'shoulda-matchers', '4.0.0'
   gem 'rails-controller-testing'
   gem 'simplecov', require: false
+
+  # Add Junit formatter for rspec
+  gem "rspec_junit_formatter"
```

Comment:
Is this because Azure needs this format?

Does it affect the readability of the spec output locally?

Reply:
Azure DevOps needs Junit format to parse the tests for CI. It doesn’t alter the output locally as it has to be called explicitly with the RSpec format option.